### DISCUSSION TITLE
Reaper pre-release with atom:Path support.

### DISCUSSION THREAD

**Discussion by 38github** [mikeoliphant/neural-amp-modeler-lv2#25] [Show and tell]
https://forum.cockos.com/showthread.php?t=279338

**Comment by mikeoliphant** [comment]
Nice! Have you tried it?

**Reply by 38github** [reply]
It does not show any type of loading function. Only the two sliders.

**Reply by mikeoliphant** [reply]
That's what I saw, too - both with the neural-amp-modeler-lv2 plugin and with other plugins I tried that use atom:Path.

**Comment by mikeoliphant** [comment]
Turns out I didn't have the correct release. You need to be sure to get the "v6.79+dev0522" prerelease version.

There was an unrelated crashing issue with Reaper. It looks like a Reaper bug, but I think I've managed to work around it.

It is working well in Reaper for me, now (on Windows - haven't tried Linux).

**Comment by mikeoliphant** [comment]
Spoke too soon - still have issues.

**Comment by mikeoliphant** [comment]
Ok - all good now with my latest checkin.

**Comment by 38github** [comment]
It works great with the _dev_ version!

**Comment by 38github** [comment]
I am closing this topic now that Reaper has this support.
